internal/prompt/prompt.go

```diff
@@ -18,14 +19,17 @@ type EventDispatcher interface {
 	EventWithLabel(category, action string, label string, dim ...*dimensions.Values)
 }
 
-// Prompter is the interface used to run our prompt from, useful for mocking in tests
+// Prompter is the interface used to run our prompt from
 type Prompter interface {
 	Input(title, message string, defaultResponse *string, flags ...ValidatorFlag) (string, error)
 	InputAndValidate(title, message string, defaultResponse *string, validator ValidatorFunc, flags ...ValidatorFlag) (string, error)
 	Select(title, message string, choices []string, defaultResponse *string) (string, error)
-	Confirm(title, message string, defaultChoice *bool) (bool, error)
+	Confirm(title, message string, defaultChoice *bool, forcedChoice *bool) (bool, error)
```

Comment:
This is only setting the forced value for confirm. The story is about all prompts. Anywhere where we handle defaultResponse and thus non-interactive we should also handle force.